### PR TITLE
Colorbalance : fix ASC CDL implementation

### PR DESCRIPTION
**Pull request by aurelienpierre**
I did a mistake in the ASC CDL (slope/offset/power) implementation, using `power = (1 / (user_input + 1))` (as the lift/gamma/gain does).

The standard implementation is `power = user_input`, which is corrected here as `power = (2 - user_input)` to be consistent with the user input scaling (between -1 and 1).

This will affect existing editings and needs to be corrected ASAP, although the visual difference is not dramatic. It also makes the auto-neutralization more accurate and the settings easier to control.

### CODE REVIEW

File: src/iop/colorbalance.c [hunk 1]
```diff
@@ -1120,13 +1139,6 @@ static void apply_gain_auto(dt_iop_module_t *self)
 }
 #endif
 
-static void disable_colorpick(struct dt_iop_module_t *self)
```
Comment (TurboGit) [inline src/iop/colorbalance.c]: @aurelienpierre : I suppose all the changes below are not expected. right?

Comment (TurboGit) [reply src/iop/colorbalance.c]: Yes, "git reset -p HEAD^" but this is not on the top commit so you also need to play with git rebase -i I suppose. I can handle this in less than a minute, so I can do it if you don't feel messing with Git. Just let me know.

Comment (aurelienpierre) [reply src/iop/colorbalance.c]: Ok I let you do it. Thank you !

Comment (TurboGit) [reply src/iop/colorbalance.c]: Done, please double check that I haven't mess anything. Thanks.

Comment (aurelienpierre) [reply src/iop/colorbalance.c]: Looks good. I will test later.

**Pull request closed without merging**